Geometry/HcalTestBeamData/plugins/DDEcalPreshowerAlgoTB.cc

```diff
@@ -138,7 +138,7 @@ void DDEcalPreshowerAlgoTB::doLayers(DDCompactView& cpv) {
 
 void DDEcalPreshowerAlgoTB::doWedges(DDCompactView& cpv) {
   LogDebug("HCalGeom") << "DDEcalPreshowerAlgoTB::debug : doWedges()";
-  int nx(0), ny(0), icopy(0), icopyt(0);
+  int icopy(0), icopyt(0);
```

Comment:
@smuzaffar - thanks for looking into it. It looks like it is a legacy DDD. Is it the same in the DD4hep algorithms?

Reply:
@ianna , I think DD4Hep algorithms are fine, at least I do not see warnings from them in https://cmssdt.cern.ch/SDT/cgi-bin/showBuildLogs.py/el8_amd64_gcc11/www/thu/13.0.CLANG-thu-23/CMSSW_13_0_CLANG_X_2023-01-12-2300